src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -51,11 +63,21 @@ public void GetAccessControl_FileInfo_InvalidArguments()
         [Fact]
         public void GetAccessControl_FileInfo_ReturnsValidObject()
         {
-            FileInfo fileInfo = new FileInfo("\\");
+            string directory = Path.GetTempPath() + Guid.NewGuid();
+            string file = Path.Combine(directory, Guid.NewGuid() + ".txt");
 
-            FileSecurity fileSecurity = FileSystemAclExtensions.GetAccessControl(fileInfo);
+            using (new TempDirectory(directory))
+            {
+                using (new TempFile(file))
```

Comment:
Also note that the outer `using` doesn't need curlies if it's immediately outside the inner `using`